src/Form/Control/TreeItemSelector.php

```diff
@@ -80,6 +80,9 @@ protected function init(): void
     {
         parent::init();
 
+        // TreeItemSelector use Vue components.
```

Comment:
comments like this should be dropped, yes, this component use vue :)

Reply:
Property not use now